pkg/agent/controller/traceflow/packetin.go

```diff
@@ -99,18 +102,25 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1alpha1.Tracefl
 	}
 	c.runningTraceflowsMutex.RUnlock()
 	if !exists {
-		return nil, nil, fmt.Errorf("Traceflow for dataplane tag %d not found in cache", pktIn.Data.Ethertype)
+		return nil, nil, nil, fmt.Errorf("Traceflow for dataplane tag %d not found in cache", pktIn.Data.Ethertype)
 	}
 
+	var capturedPacket *crdv1alpha1.Packet
 	if tfState.liveTraffic && firstPacket {
 		// Uninstall the OVS flows after receiving the first packet, to
 		// avoid capturing too many matched packets.
 		c.ofClient.UninstallTraceflowFlows(tag)
+		// Report the captured dropped packet, if the Traceflow is for
+		// the dropped packet only; otherwise only the sender reports
+		// the first captured packet.
+		if tfState.isSender || tfState.droppedOnly {
```

Comment:
If droppedOnly is set and the packet is dropped on receiver, could the sender and the receiver override the field that have been written by the other one? Is it expected?

Reply:
Not really. For droppedOnly, I changed to flow to send to controller only when the packet is dropped, so only one of sender or receiver should come to here.